cpp/test/Slice/utf8BOM/test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) ZeroC, Inc.
+
+import os
+import shutil
+
+from Util import ClientTestCase, SliceTranslator, TestSuite, platform, Linux
+
+
+class SliceUtf8BomTestCase(ClientTestCase):
+    def runClientSide(self, current):
+        test_file_name = "Test.ice"
+        slice2cpp = SliceTranslator("slice2cpp", quiet=True)
+
+        # Temporary output directory for 'slice2cpp to use.
```

Comment:
Why do we need to create the output directory? I was thinking slice2cpp shouldn't generate any code here as the Slice file has errors.

Reply:
I added it just to make sure that the tests will clean up after themselves no matter what.
Although you're right that under normal circumstances, nothing will be generated.

We also create this tmp folder in the `errorDetection` tests, which is in the same situation: https://github.com/zeroc-ice/ice/blob/86dfedab1473ae2d03d65309ac78b1400229d86a/cpp/test/Slice/errorDetection/test.py#L16-L19